java/src/jmri/jmrit/roster/Roster.java

```diff
@@ -1152,6 +1152,7 @@ public void addRosterGroup(RosterGroup rg) {
             return;
         }
         this.rosterGroups.put(rg.getName(), rg);
+        log.debug("firePropertyChange Roster Groups model:" + rg.getName()); // test for panel redraw after duplication
```

Comment:
Can you change this from: ```model:" + rg.getName()``` to ```model: {}", rg.getName()```?

[What is the fastest way of (not) logging?](http://www.slf4j.org/faq.html#logging_performance) explains why.